### DISCUSSION TITLE
Better support for cameras with multiple binning support

### DISCUSSION THREAD

**Discussion by Bladerunner1452** [Stellarium/stellarium#1735] [Ideas]
At first: Many thanks for this really great planetarium program! I use it standalone as well as in combination with my Skywatcher mount (controlled by EQMOD) and APT (Astro Photography Tool) to point the mount to objects centered in Stellarium; this really works great!
But I have a suggestion for better support of cameras which can switch between multiple binnings:
I always use Stellarium for planning my photo sessions. In the upper right corner (when I use the "sensor view") I can easily switch between cameras (resp. sensors), telescopes (incl. objective lenses), barlows and rotations angles. But for switching the binning I have to configure the camera/sensor again for each different binning (I use the FOV and the x/y scale (arcsecs/Pixel) to decide, which lens/telescope to use AND the arcsecs/pixel to decide, which binning to use).
It would be much better to handle, if I could enter a list of supported binnings for my cameras and could switch the binning in the same way as the other details. This could be implemented completely compatible to the actual manner, because a list also can contain only one element. So it could be entered for example as comma separated list.
With kind regards
Wolfgang

**Comment by gzotti** [comment]
IMHO configuring your camera 3 times for three binning configurations is not too complicated.
